controllers/message_answer.go

```diff
@@ -493,3 +493,41 @@ func (c *ApiController) GetAnswer() {
 	}
 	c.ResponseOk(answer)
 }
+
+// GetQuestionAnswer
```

Comment:
Use existing `GetAnswer` API, some if conditions may need to be adjusted in it to fit your case